app/Http/Controllers/Api/V1/CurrikiInteractiveVideoIntegration/Kaltura/KalturaGeneratedAPIClientController.php

```diff
@@ -46,10 +46,8 @@ public function __construct(KalturaConfiguration $kC, KalturaClient $kClient, Ka
     }
 
     /**
-     * Method       getMediaEntryList
-     * Description  Use Kaltura Session to get the api token
-     * Purpose      To get those media list, which do not have any 'Entitlement Enforcement/Permission Category'
-     * Usage        Inside H5p Curriki Interactive Video
+     * getMediaEntryList
```

Comment:
Need to update docs as mentioned in https://beyondco.de/docs/laravel-apidoc-generator/getting-started/documenting-your-api

Reply:
Done!